src/scripts/services/user-service.ts

```diff
@@ -109,36 +111,61 @@ export class OktaUserService {
   };
 
   // eslint-disable-next-line functional/functional-parameters
-  readonly listUsers = (): TE.TaskEither<string, readonly User[]> => {
+  readonly listUsers = (): TE.TaskEither<string, readonly User[]> =>
+    // eslint-disable-next-line functional/no-this-expression
+    this.privateListUsers(TE.right(this.client));
+
+  readonly listUsersInGroup = (
+    group: O.Option<Group>
```

Comment:
This shouldn't be an option.

We should attempt to prevent callers from calling us incorrectly. None is never a safe option here, so should be made impossible by the type system.

Reply:
Fold was moved up to the `list-users.ts` level instead